decayAmplitude/waveDescription.cc

```diff
@@ -868,23 +1004,117 @@ waveDescription::mapMassDependenceType(const Setting* massDepKey)
 	else if (massDepType == "rhoPrime")
 		massDep = createRhoPrimeMassDep();
 	else if (massDepType == "binned") {
-		const libconfig::Setting* bounds = rpwa::findLibConfigList(*massDepKey, "bounds" , false);
+		const Setting* bounds = findLibConfigList(*massDepKey, "bounds" , false);
 		if (not bounds) {
-			printErr << "no bounds given for binned mass dependence." << std::endl;
+			printErr << "no bounds given for binned mass dependence." << endl;
 			throw;
 		}
 		const int length = bounds->getLength();
 		if (length != 2) {
-			printErr << "bounds do not have the required length (expected: 2, found: " << length << ")." << std::endl;
+			printErr << "bounds do not have the required length (expected: 2, found: " << length << ")." << endl;
 			throw;
 		}
 		const double mMin = (*bounds)[0];
 		const double mMax = (*bounds)[1];
 		if (mMin > mMax) {
-			printErr << "bounds are not ordered: mMin(" << mMin << ") > mMax(" << mMax << ")." << std::endl;
+			printErr << "bounds are not ordered: mMin(" << mMin << ") > mMax(" << mMax << ")." << endl;
 			throw;
 		}
-		massDep = createbinnedMassDependence(mMin, mMax);
+		massDep = createBinnedMassDependence(mMin, mMax);
+	}
+	else if (massDepType == "sawtooth") {
```

Comment:
since the checking of the bounds is the same for the binned and sawtooth cases code should be unified